.changeset/rare-cobras-poke.md

```diff
@@ -0,0 +1,5 @@
+---
+'@rocket.chat/gazzodown': major
```

Comment:
I don't think that fixing a behavior of a component in one package is enough to release a new major version of it.

Reply:
my bad, I am not fully comfortable with this change set structure so I may set this as a major by mistake